snmpwalk/README.md

```diff
@@ -2,15 +2,21 @@
 
 ## Overview
 
-Get metrics from SNMP walk service in real time to:
+This integration provides insight into SNMP walk states in real time to:
 
-- Visualize and monitor SNMP walk states
-- Be notified about SNMP walk failovers and events.
+- Visualize and monitor SNMP walk states.
+- Be notified about SNMP walk failovers.
 
 ## Setup
 
 The SNMP walk check is not included in the [Datadog Agent][2] package, so you need to install it.
 
+You can also use the [snmpwalk][11] command as an alternative to this integration:
```

Comment:
```suggestion
You can also use the [`snmp walk`][11] command as an alternative to this integration:
```
I followed the link and it looked like the use of this command was for a fairly different use case. I'm not super familiar with it, so I'll defer to you - just want to make sure that the link will help users understand what's happening rather than confuse them about what they're trying to accomplish when they're trying to decide whether they want to use the integration or the command.

Reply:
Yeah makes sense, I reviewed this with the Agent Integrations team and this particular integration is ~ 7 yrs old and not really maintained anymore, but we don't want to remove it completely so wanted users to know they can use the Agent command as well or in replace of this.